docs/7.x/requirements.md

```diff
@@ -134,10 +134,10 @@ These ports are used for Cluster operation and should be open between cluster no
 | 3012                    | TCP                | HTTPS                | all         | all         | Gravity RPC agent                        |
 
 !!! note "Source/Destination Legend"
-  * all - Any node which is a member of the cluster
-  * ext - Any source outside the cluster
-  * localhost - The port is only used within the host where the request started
-  * controllers - Nodes which are designated "controller" (aka "master") role
+    * all - Any node which is a member of the cluster
+    * ext - Any source outside the cluster
+    * localhost - The port is only used within the host where the request started
+    * controllers - Nodes which are designated "controller" (aka "master") role
```

Comment:
nit: Maybe use `control plane` instead of `controller` when describing a controller node?

Reply:
This change isn't pertinent to the intended diff under review.  As mentioned in the description, please use the following link to review (at least until #1872 lands and this is rebased):

https://github.com/gravitational/gravity/pull/1873/files/4bb9592c8fcd3f3f47e7308c3fee0bc3d9bc90f9..ba29d722e11956709c016064993b2886ac850c47

We'll carry this discussion forward at https://github.com/gravitational/gravity/pull/1872#discussion_r455276693